### ISSUE TITLE
The current data structure has changed and the data before parsing binlog will go wrong

### ISSUE THREAD

**Issue by hardcao** [noplay/python-mysql-replication#220]

**Comment by hardcao** [comment]
For example: the current time is 6 points, and now the binlog parsing time is 5 points, at 6 points to the table structure added a field, at this time binlog parsing will have problems

**Comment by baloo** [comment]
Yes and this is an unsolved problem for now. #176 added a potential solution for this.
